Classes/Queue/DoctrineQueue.php

```diff
@@ -114,7 +114,11 @@ public function setUp(): void
                 $createDatabaseStatement = "CREATE TABLE IF NOT EXISTS {$this->connection->quoteIdentifier($this->tableName)} (id INTEGER PRIMARY KEY AUTO_INCREMENT, payload LONGTEXT NOT NULL, state VARCHAR(255) NOT NULL, failures INTEGER NOT NULL DEFAULT 0, scheduled DATETIME DEFAULT NULL) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB";
         }
         $this->connection->exec($createDatabaseStatement);
-        $this->connection->exec("CREATE INDEX state_scheduled ON {$this->connection->quoteIdentifier($this->tableName)} (state, scheduled)");
+        try {
+            $this->connection->exec("CREATE INDEX state_scheduled ON {$this->connection->quoteIdentifier($this->tableName)} (state, scheduled)");
+        } catch (Exception $e) {
+            // See https://dba.stackexchange.com/questions/24531/mysql-create-index-if-not-exists
```

Comment:
IMO it's a dangerous pattern to just silently ignore a whole class of exceptions.. IMO we should just catch the exact exception class for the described case and (in the long run) allow the setup to report back